test/unit/org/apache/cassandra/cql3/validation/operations/SelectLimitTest.java

```diff
@@ -39,6 +40,7 @@ public static void setUpClass()
 
         StorageService.instance.setPartitionerUnsafe(ByteOrderedPartitioner.instance);
         DatabaseDescriptor.setPartitionerUnsafe(ByteOrderedPartitioner.instance);
+        DatabaseDescriptor.setDefaultSecondaryIndex(CassandraIndex.NAME);
```

Comment:
Does this line force the test to run with always with that index flavor? If it were so an assume would spare some CI cycles.

Reply:
BOP is incompatible with SAI.

Only one test uses indexing here, though, so changed only that test to `assumeLegacySecondaryIndex()`.